escoba/src/main.js

```diff
@@ -0,0 +1,46 @@
+import '@babel/polyfill'
+import 'mutationobserver-shim'
+import Vue from 'vue'
+import './plugins/bootstrap-vue'
+import App from './App.vue'
+import router from './router'
+import store from './store'
+import './registerServiceWorker'
+import axios from 'axios'
+import VueAxios from 'vue-axios'
+import BootstrapVue from 'bootstrap-vue'
+import 'bootstrap/dist/css/bootstrap.css'
+import 'bootstrap-vue/dist/bootstrap-vue.css'
+import 'vue-awesome/icons/trophy'
+import 'vue-awesome/icons/screwdriver'
+import 'vue-awesome/icons/hammer'
+import 'vue-awesome/icons/hand-spock'
+import 'vue-awesome/icons/child'
+import 'vue-awesome/icons/brands/bitcoin'
+import 'vue-awesome/icons/brands/stack-overflow'
+import 'vue-awesome/icons/poo'
+import VueSwing from 'vue-swing'
+
+import Icon from 'vue-awesome/components/Icon'
+
+Vue.component('v-icon', Icon)
+
+Vue.component('vue-swing', VueSwing)
+
+Vue.use(BootstrapVue)
+
+Vue.config.productionTip = false
+
+store.dispatch('loadGameData').then(() => {
+  router.beforeEach((to, from, next) => {
+    next()
+  })
+})
+
+new Vue({
+  router,
+  store,
+  axios,
```

Comment:
axios and vue-axios?  from what i have seen it's generally used. wasn't too sure.  this article gives more info.  doesn't seem like it would hurt either way.  could figure out the config i'm sure. https://dev.to/heftyhead/lets-talk-about-an-unnecessary-but-popular-vue-plugin-1ied